### PR TITLE
add qdrant blog post page

### PR DESCRIPTION
**Pull request by nastyapash**

### CODE REVIEW

**Comment by nastyapash** [comment]
@trean The review block on a blue background was updated in Figma. Also, it is completely different in the mobile version. 
Should I change this block (photo) in .md file? 
Do I need to make this block different for mobile version and desktop version as in Figma, or leave one option with a blue background?

**Comment by nastyapash** [comment]
@trean  The review block on a blue background was updated in Figma. Also, it is completely different in the mobile version.
Should I change this block (image) in .md file?
Do I need to make this block different for mobile version and desktop version as in Figma, or leave one option with a blue background?

File: qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-blog-post.scss [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    &__header {
+      max-width: rem(920);
+      margin: 0 auto;
+      padding-top: $spacer * 5;
+      padding-bottom: $spacer * 5;
+    }
+
+    &__title {
+      font-size: $spacer * 3.5;
+      line-height: rem(57);
```
Comment (trean) [inline qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-blog-post.scss:113]:
```suggestion
      line-height: rem(67);
```

The `line-height` in design appears to be set to 67px. Now the header looks a bit too compact.

Screenshot 2024-04-10 at 12 32 13

File: qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-blog-post.scss [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+    &-links {
+      display: flex;
+      justify-content: flex-end;
+      align-items: center;
+      gap: $spacer * 2;
+    }
+  }
+
+  &__content {
```
Comment (trean) [inline qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-blog-post.scss:65]: Could you please add an underline effect to the links on hover?

**Review by trean — CHANGES_REQUESTED** [review]
Thank you, looks good! I left a couple of comments.

File: qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-blog-post.scss (hunk 2 above)
Comment (nastyapash) [reply qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-blog-post.scss:65]: done

File: qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-blog-post.scss (hunk 1 above)
Comment (nastyapash) [reply qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-blog-post.scss]: done

**Review by trean — APPROVED** [review]

**Pull request merged**